examples/performance/main.rs

```diff
@@ -94,7 +94,6 @@ trait Renderer: Drop {
 struct GFX {
     dimension: i16,
     window: gfx_window_glutin::Window,
-    surface: gfx_window_glutin::Surface,
```

Comment:
I'm not totally sure if dropping it early is safe, haven't looked into details about the required lifetime of `Surface`s but it's on my TODO. But for GL it shouldn't be a problem iirc.